### PR TITLE
RAC-4485 Remove Generated Files

### PR DESCRIPTION
**Pull request by BrentHiggins**
* Updated version of the PR to remove generated files

### CODE REVIEW

**Comment by BrentHiggins** [comment]
@jimturnquist @hohene

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+# Ignore these generated files so they dont get checked in again
+config.ini
+generate-sol-log.sh
+post-deploy.sh
```
Comment (johren) [inline .gitignore:5]: Missing pre-deploy.sh

**Review by johren — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (johren) [reply .gitignore:5]: And vnc-record.sh

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+test.sh
+.nvmrc
```
Comment (johren) [inline .gitignore]: .nvmrc no longer needed since .nvmrc.in was removed in a previous PR.

**Review by jimturnquist — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+# Ignore these generated files so they dont get checked in again
+config.ini
+generate-sol-log.sh
+post-deploy.sh
+test.sh
+pre-deploy.sh
+vnc-record.sh
```
Comment (johren) [inline .gitignore]: Actually, it is an underscore rather than a dash.  I got it wrong in the comment.  
vnc_record.sh

**Comment by panpan0000** [comment]
Thank you to doing that !
@PengTian0  for awareness.

**Comment by PengTian0** [comment]
:+1:

**Review by johren — APPROVED** [review]

**Pull request merged**